csrc/quantization/gguf/mmq.cuh

```diff
@@ -0,0 +1,659 @@
+// MMQ
```

Comment:
Ditto, I assume it is https://github.com/ggerganov/llama.cpp/blob/dd07a123b79f9bd9e8a4ba0447427b3083e9347a/ggml/src/ggml-cuda/mmq.cuh